project.clj

```diff
@@ -131,14 +134,12 @@
                                      ;; (and shipped with this refactor-nrepl):
                                      :plugins      [[cider/cider-nrepl "0.24.0"]]}
 
-             :parallel-eastwood     {:jvm-opts ["-Dformatting-stack.eastwood.parallelize-linters=true"]}
```

Comment:
Eastwood already parallelizes linters now

(f-s' own testing aided that decision!)